Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -518,25 +515,55 @@ - (void)loadVungleNativeAdForParameters:(id<MAAdapterResponseParameters>)paramet
     return clickableViews;
 }
 
-- (BannerSize)adSizeFromAdFormat:(MAAdFormat *)adFormat
+- (VungleAdSize *)adSizeFromAdFormat:(MAAdFormat *)adFormat
+                          parameters:(id<MAAdapterParameters>)parameters
 {
-    if ( adFormat == MAAdFormat.banner )
-    {
-        return BannerSizeRegular;
-    }
-    else if ( adFormat == MAAdFormat.leader )
+    BOOL isAdaptiveBanner = [parameters.localExtraParameters al_boolForKey: @"adaptive_banner"];
+    NSNumber *customWidth = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_width"];
+    NSNumber *customHight = [parameters.localExtraParameters al_numberForKey: @"adaptive_banner_height"];
+    
+    if ( isAdaptiveBanner )
     {
-        return BannerSizeLeaderboard;
+        return [VungleAdSize VungleAdSizeFromCGSize:(CGSizeMake(customWidth.floatValue, 0))];
```

Comment:
I think we can remove this adaptive part.

Reply:
Yes. I removed adaptive case. Thanks,